### DISCUSSION TITLE
how it works (so you lazy bastards can try to fix)

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#98] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
it literally just mimics the normal menu and sends the off signal. Literally, that's all. now we need an _acual_ exploit which will _actually_ do something cheaty cheaty. thing is, I'm too lazy so figure it out yourselves. you can, though, still use the brutal method that I invented. 
paste this into your bookmark: `javascript:(function(){window.onbeforeunload=function(){return '';}})();`. get ready to quickly visit the site, crash goguardian until the page loads, press the bookmark, and just ignore when it tries to close the page.

**Comment by ghost** [comment]
but the thing is javascript is blocked

**Reply by ghost** [reply]
k then you're fucked

**Reply by ghost** [reply]
nah ppl are making new exploit

**Reply by ghost** [reply]
@ttvbeghhf good for you

**Reply by synapt0** [reply]
my school has dyno and securly

**Reply by LogA1Cyber** [reply]
my school just fixed the bookmark hack, any way on making it work again?

**Reply by md4r2** [reply]
@NotJantix bruh lol

**Reply by synapt0** [reply]
idk bout no goguardian but before the update happened I was able to unblock stuff too even with dyno and securly

**Comment by The-Man-Is-Me** [comment]
Does anyone actually know why the update broke the exploit???

**Reply by ghost** [reply]
it blocked javascript from working

**Reply by synapt0** [reply]
idk sumn in the 106.x... sumn update broke it

**Reply by amelia-bartlett** [reply]
> it blocked javascript from working

Yeah basically they protected privileged pages better

**Comment by Alex2Mayhem** [comment]
> it literally just mimics the normal menu and sends the off signal. Literally, that's all. now we need an _acual_ exploit which will _actually_ do something cheaty cheaty. thing is, I'm too lazy so figure it out yourselves. you can, though, still use the brutal method that I invented. paste this into your bookmark: `javascript:(function(){window.onbeforeunload=function(){return '';}})();`. get ready to quickly visit the site, crash goguardian until the page loads, press the bookmark, and just ignore when it tries to close the page.

What site?

**Reply by ghost** [reply]
the site you want to visit

**Comment by Alex2Mayhem** [comment, marked as answer]
Here is another way to run the JS use this website and remove everything else except the script.js put the code in there https://www.programiz.com/html/online-compiler/

**Reply by ghost** [reply]
wym like the bookmark code?
